lib/ContentHelpers.php

```diff
@@ -181,9 +181,23 @@ public static function extractDocumentGroups($documentGroupsField)
     }
 
     // Use custom thumbnail if one is set, otherwise default to hero image.
-    public static function getProgrammeThumbnail($entry)
+    // @TODO: Remove one new hero images are all in place
+    public static function getFundingProgrammeThumbnailUrl($entry)
     {
-        $heroImage = Images::getHeroImageField($entry->hero);
+        $heroImage = Images::extractImage($entry->heroImage);
```

Comment:
This change has been causing me issues with the live CMS where I chose a strategic programme to go with a blogpost and then got `Call to a member function one() on null` – traced it back to the `extractImage` call here, which assumes there's an image to call `one()` against. Maybe we can fix this, might introduce other errors where we're assuming things have heroes?